src/main/java/Duke.java.orig

```diff
@@ -0,0 +1,331 @@
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+<<<<<<< .merge_file_a00820
+import java.util.ArrayList;
+=======
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+>>>>>>> .merge_file_a24788
+import java.util.Scanner;
+
+public class Duke {
+	static ArrayList<Task> memory;
+	static String line = "____________________________________________________________";
+	
+<<<<<<< .merge_file_a00820
+	public static void main(String[] args) throws IOException {
+=======
+	public static void main(String[] args) throws IOException, ParseException {
+>>>>>>> .merge_file_a24788
+		memory = new ArrayList<>();
+		
+		loadSave();
+		
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        
+		System.out.println(line);
+		System.out.println(" Hello! I'm Duke");
+		System.out.println(" What can I do for you?");
+		System.out.println(line);
+		System.out.println();
+		
+		Scanner sc = new Scanner(System.in);
+		while (sc.hasNext()) {
+			String rawInput = sc.nextLine();
+			
+			if (rawInput.toLowerCase().equals("bye")) {
+				System.out.println(line);
+				System.out.println(" " + "Bye. Hope to see you again soon!");
+				System.out.println(line);
+				System.out.println();
+				break;
+			} else if (rawInput.toLowerCase().equals("list")) {
+				System.out.println(line);
+				System.out.println(" " + "Here are the tasks in your list:");
+				list();
+				System.out.println(line);
+				System.out.println();
+			} else if (rawInput.toLowerCase().startsWith("done")) {
```

Comment:
I like the way you use .toLowerCase here to allow for the user to type instructions that may not necessarily be all lower-case which allows user's to interact with duke with less restrictions when it comes to typing out of instructions